README.md

```diff
@@ -13,8 +13,7 @@ Details of our interpretation of OAuth are provided in
 
 The application has Rake tasks to create new users and client applications.
 
-To create a new client application to which Sign-on-o-Tron will provide sign-on
-services:
+To create a new client application which will provide sign-on services:
```

Comment:
This new wording isn't quite correct. It's suggesting that the client application will provide the sign-on services, when it's actually this application that provides them, and the client application uses them.